tests/sast/registry_tests/test_registry.py

```diff
@@ -1,24 +1,53 @@
+from checkov.runner_filter import RunnerFilter
 from checkov.sast.consts import SastLanguages
 from checkov.sast.checks.base_registry import Registry
 import pathlib
 import os
 import unittest
 
+python_rule = {'id': 'CKV_SAST_1', 'message': 'module setting superuser port', 'severity': 'INFO',
+                       'languages': ['python'], 'metadata': {'name': 'superuser port',
+                                                             'cwe': 'CWE-289: Authentication Bypass by Alternate Name'},
+                       'patterns': [{'pattern': 'set_port($ARG)'}, {'metavariable-comparison': {
+                           'metavariable': '$ARG', 'comparison': '$ARG < 1024'}}]}
 
+
+java_rule = {'id': 'seam-log-injection', 'message': 'Seam Logging API support an expression language to introduce bean property to log messages. The expression language can also be the source to unwanted code execution. In this context, an expression is built with a dynamic value. The source of the value(s) should be verified to avoid that unfiltered values fall into this risky code evaluation.',
+             'severity': 'ERROR', 'languages': ['java'], 'metadata': {
+                'name': 'seam log injection',
+                'cwe': "CWE-95: Improper Neutralization of Directives in Dynamically Evaluated Code ('Eval Injection')",
+                'owasp': 'A03:2021 - Injection'},
+                'patterns': [{'pattern': '$LOG.$INFO($X + $Y,...)'},
+                    {'pattern-either': [{'pattern-inside': 'import org.jboss.seam.log.Log\n...\n'},
+                          {'pattern-inside': 'org.jboss.seam.log.Log $LOG = ...\n...\n'}]},
+                          {'metavariable-regex': {'metavariable': '$INFO',
+                                                  'regex': '(debug|error|fatal|info|trace|warn)'}}]}
 class TestRegistry(unittest.TestCase):
```

Comment:
It's probably too late now to comment on as this was done previously, but what do you think on stopping usage of `unittest` in SAST and instead use `pytest`?
Currently we have a wierd mix of both in checkov and I believe the intention is to eventually migrate to pytest, so at least when we are writing something new let's do it like that from the get go.

I don't want you to change it now though, just a thought for the future :)